yt/chyt/controller/internal/jupyt/api.go

```diff
@@ -16,6 +17,33 @@ type GetEndpointResult struct {
 	JobID       yt.JobID       `yson:"job_id"`
 }
 
+func getEndpointPortByJob(ctx context.Context, job yt.JobStatus, ytc yt.Client) (port int, err error) {
+	path := ypath.Path("//sys/exec_nodes").
+		Child(job.Address).
+		Child("/orchid/exec_node/job_controller/active_jobs/").
```

Comment:
First, it doesn't look right. There should be no `/` in the beginning and at the end of an argument.
Second, you have many `.Child()` here. The correct method to use is `.JoinChild` to join multiple children.